play/src/front/Phaser/Game/GameScene.ts

```diff
@@ -518,48 +517,6 @@ export class GameScene extends DirtyScene {
                 }
             }
         }
-
-        for (const [itemType, objectsOfType] of this.objectsByType) {
-            // FIXME: we would ideally need for the loader to WAIT for the import to be performed, which means writing our own loader plugin.
-
-            let itemFactory: ItemFactoryInterface;
-
-            switch (itemType) {
-                case "computer": {
-                    const module = await import("../Items/Computer/computer");
-                    itemFactory = module.default;
-                    break;
-                }
-                default:
-                    continue;
-                //throw new Error('Unsupported object type: "'+ itemType +'"');
-            }
-
-            itemFactory.preload(this.load);
-            this.load.start(); // Let's manually start the loader because the import might be over AFTER the loading ends.
-
-            this.load.on("complete", () => {
-                // FIXME: the factory might fail because the resources might not be loaded yet...
-                // We would need to add a loader ended event in addition to the createPromise
-                this.createPromiseDeferred.promise
-                    .then(async () => {
-                        itemFactory.create(this);
-
-                        const roomJoinedAnswer = await this.connectionAnswerPromiseDeferred.promise;
-
-                        for (const object of objectsOfType) {
-                            // TODO: we should pass here a factory to create sprites (maybe?)
-
-                            // Do we have a state for this object?
-                            const state = roomJoinedAnswer.items[object.id];
-
-                            const actionableItem = itemFactory.factory(this, object, state);
-                            this.actionableItems.set(actionableItem.getId(), actionableItem);
-                        }
-                    })
-                    .catch((e) => console.error(e));
-            });
-        }
```

Comment:
If you start removing this, there is much more we need to remove.

Actionable items are spreading in the pusher and the back too!